frontend-new/src/components/Header.vue

```diff
@@ -1,17 +1,209 @@
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-code-braces class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.code") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2"
+                                   href="https://paper.readthedocs.io/en/latest/"
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-book-open class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.docs") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2" href="https://papermc.io/javadocs"
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-language-java class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.javadocs") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2" href="/"
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-puzzle class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.hangar") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2" href="https://papermc.io/downloads"
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-download-circle class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.downloads") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2" href="https://papermc.io/community"
+                                   hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-account-group class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.community") }}
+                                </a>
+                                <a
+class="flex items-center rounded-md px-6 py-2"
+                                   href="https://hangar-auth.benndorf.dev/" hover="text-primary-100 bg-primary-50">
+                                    <icon-mdi-key class="mr-3 text-[1.2em]"/>
+                                    {{ t("nav.hangar.auth") }}
+                                </a>
+                            </div>
+                        </PopoverPanel>
+                    </transition>
+                </Popover>
+
+                <div class="site-logo mr-4 h-60px flex items-center">
+                    <img alt="Hangar Logo" src="../../public/logo.svg" class="h-50px object-cover"/>
```

Comment:
can we use the import alias here? I think I configured @. relative paths like that quickly become a pain